### PR TITLE
Backport old lookups

### LINKED ISSUE #3126: Port deprecated lookup extension ONLY for 0.9.1
As per https://groups.google.com/d/msg/druid-development/x8FQeSPvWtI/zetL6s1nBwAJ this is to port the old extension to be distributed with 0.9.1

### PR DESCRIPTION
**Pull request by drcrallen**
Fixes #3126

### CODE REVIEW

File: docs/content/development/extensions-core/namespaced-lookup.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+layout: doc_page
+---
+
+<div class="note caution">
+This extension has been deprecated as of 0.9.1.
+</div>
+
+# Transitioning to lookups-cached-global
+
+New users should reference the exteion [lookups-cached-global](lookups-cached-global.html)
```
Comment (gianm) [inline docs/content/development/extensions-core/namespaced-lookup.md]: extension (sp)

File: docs/content/development/extensions-core/namespaced-lookup.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+and change it to
+
+```json
+"extractionFn":{
+  "type":"registeredLookup",
+  "lookup":"some_uri_lookup"
+}
+```
+
+Once all your queries are using the "new" extraction syntax, you can remove the static configuration and reboot those nodes, or remove the static configuration when you next perform normal maintenance.
```
Comment (gianm) [inline docs/content/development/extensions-core/namespaced-lookup.md]: remove static configuration, and remove the druid-namespace-lookup extension

File: docs/content/development/extensions-core/namespaced-lookup.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+<div class="note caution">
+This extension has been deprecated as of 0.9.1.
+</div>
+
+# Transitioning to lookups-cached-global
+
+New users should reference the exteion [lookups-cached-global](lookups-cached-global.html)
+
+## Moving to cluster wide configuration
```
Comment (gianm) [inline docs/content/development/extensions-core/namespaced-lookup.md]: I think we should link this doc from lookups.md and lookups-cached-global.md, with language like "If you are currently using the druid-namespace-lookup extension, please see namespaced-lookup.html for instructions on migrating to the new lookup framework. The druid-namespace-lookup extension has been deprecated and will be removed in a future version of Druid."

File: extensions-core/namespace-lookup/pom.xml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.druid.extensions</groupId>
+  <artifactId>druid-namespace-lookup</artifactId>
+  <name>druid-namespace-lookup</name>
+  <description>Extension to rename Druid dimension values using namespaces</description>
+
+  <parent>
+    <groupId>io.druid</groupId>
+    <artifactId>druid</artifactId>
+    <version>0.9.1-rc2-SNAPSHOT</version>
```
Comment (gianm) [inline extensions-core/namespace-lookup/pom.xml]: this version probably needs updating

File: docs/content/development/extensions-core/namespaced-lookup.md [hunk 5]
```diff
@@ -2,6 +2,87 @@
...
+
+This module only supports static configurations, whereas the new [lookups-cached-global](lookups-cached-global.html) extension supports only dynamic cluster configuration.
+
+Most of the configurations can be used as-is and posted to the coordinator to the appropriate endpoint. Once the cluster is upgraded to 0.9.1, the json configuration can be propogated through the cluster as follows:
```
Comment (gianm) [inline docs/content/development/extensions-core/namespaced-lookup.md]: propagated (sp)

**Comment by drcrallen** [comment]
@gianm addressed

**Comment by gianm** [comment]
👍

**Comment by fjy** [comment]
👍

**Pull request merged**